### PR TITLE
Visual changes

### PR DESCRIPTION
**Pull request by postatum**
According to page 1 from the `Raml.org / A.ml - proposed visual changes` Google Doc.

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -652,6 +689,53 @@ header{
...
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        background-color: #808080;
+        left: 0;
+        top: 0;
+        opacity: 0.1;
+      }
+      .tip-wrapper:hover {
+        cursor: pointer;
```
Comment (jstoiko) [inline css/style.scss]: Nit pick: I would remove this class as it feels a little odd to have a pointer cursor (hand) that doesn't do anything when clicking. The default cursor (arrow) might be what people expect the most in this case. Agreed?

**Review by jstoiko — COMMENTED** [review]
This looks great! Well done. 

Aside from that one minor comment I made, the only other thing is I was wondering if it would be easier (for future maintenance) to have the static editor in pure html + css as opposed to using Monaco? This may also render faster.

File: css/style.scss (hunk 1 above)
Comment (postatum) [reply css/style.scss]: Agreed. Removed.

**Comment by postatum** [comment]
> I was wondering if it would be easier (for future maintenance) to have the static editor in pure html + css as opposed to using Monaco? This may also render faster.

It would probably render faster, but I doubt it would be easier to maintain.

I think so because here are the possible solutions I can think of:
- Find a replacement lib. It should either simply draw editor without any functionality or just a simplified version if Monaco editor. I didn't find such libs after a few search attempts. 
- Write everything with pure HTML+CSS. In this case it would probably load faster, but would be super-hardcoded as each piece of the RAML example would have to be styled and positioned statically.
- Write some JS lib to draw the very simple editor HTML+CSS. In this case other than the drawing logic itself, we would also have to write a RAML tokenizer to know which piece of RAML is which and how to style/position it.

File: index.html [hunk 2]
```diff
@@ -32,14 +32,79 @@
...
+              <div class="tip-background"></div>
+              <div class="tip-text">
+                <p>Name your API, specify its version and base URL</p>
+              </div>
+            </div>
+
+            <div class="tip-wrapper" data-startline="9" data-lastslines="3">
+              <div class="tip-background"></div>
+              <div class="tip-text">
+                <p>Specify reusable types to avoid</p>
```
Comment (jstoiko) [inline index.html]: to avoid what?

Comment (postatum) [reply index.html]: duplication and redundancy :)

Thank for the catch. I've fixed it.

**Comment by jstoiko** [comment]
> Write everything with pure HTML+CSS. In this case it would probably load faster, but would be super-hardcoded as each piece of the RAML example would have to be styled and positioned statically.

I think this is something we should consider as this is not really meant to change anyway. Will merge for now but can you create an issue so we don't forget?

**Review by jstoiko — APPROVED** [review]

**Comment by postatum** [comment]
> I think this is something we should consider as this is not really meant to change anyway. Will merge for now but can you create an issue so we don't forget?

Done https://github.com/raml-org/raml-org/issues/116

**Pull request merged**